EncryptDecryptSample/app/src/androidTest/java/com/rivetz/encryptdecryptexample/ExampleInstrumentedTest.java

```diff
@@ -1,4 +1,4 @@
-package com.rivetz.r_app2;
```

Comment:
The package name must match the directory structure. It looks like you updated the package name, but need to move the file(s).